docs/static/puzzle_resources/the_dating_sim/htaccess.txt

```diff
@@ -0,0 +1,26 @@
+# https://emscripten.org/docs/compiling/WebAssembly.html#web-server-setup
+Options +MultiViews
+RemoveType .gz
+AddEncoding x-gzip .gz
+AddType application/octet-stream .data
+AddType application/octet-stream .em
+AddType application/wasm .wasm
+
+# CPU-intensive, better pre-gzip
+#AddOutputFilterByType DEFLATE application/wasm
+
+
+# Force browser to check if cache needs to be refreshed or not (304) for XHR
+# (preserve browser cache, but revalidate)
+<IfModule mod_headers.c>
+  Header set Cache-Control "max-age=0"
+</IfModule>
+
+# Disable ETags which don't play nice with mod_gzip and HTTP/1.1
```

Comment:
Do we need this block of comments?

Reply:
Almost all of the stuff in this directory was generated by renpy - I don't actually know what most of this does...

Reply:
Ah gotcha. If we remove this file, does Dating Sim still work?

IIUC, this htaccess file configures Apache web servers, which might not apply here?...unless Renpy web uses it for its own configuration?

Seems like it doesn't hurt to keep it, but if it's not needed, we should probably remove it.

Reply:
looks like it's not necessary, removed it.